gateway/gateway_test.go

```diff
@@ -875,21 +875,60 @@ func TestManagementNodeRedisEvents(t *testing.T) {
 	globalConf := config.Global()
 	globalConf.ManagementNode = false
 	config.SetGlobal(globalConf)
-	msg := redis.Message{
-		Payload: `{"Command": "NoticeGatewayDRLNotification"}`,
-	}
-	shouldHandle := func(got NotificationCommand) {
-		if want := NoticeGatewayDRLNotification; got != want {
-			t.Fatalf("want %q, got %q", want, got)
+
+	t.Run("Without signing:", func(t *testing.T) {
+		msg := redis.Message{
+			Payload: `{"Command": "NoticeGatewayDRLNotification"}`,
 		}
-	}
-	handleRedisEvent(&msg, shouldHandle, nil)
-	globalConf.ManagementNode = true
-	config.SetGlobal(globalConf)
-	notHandle := func(got NotificationCommand) {
-		t.Fatalf("should have not handled redis event")
-	}
-	handleRedisEvent(msg, notHandle, nil)
+		shouldHandle := func(got NotificationCommand) {
```

Comment:
(*) We don't if it is entered here. I think a bool should be defined, set in the func and check after the func.